_data/settings/components/print-styles.yml

```diff
@@ -0,0 +1,2 @@
+name: Print styles
```

Comment:
If there's no data component settings, is this file needed? It shows up here: https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.app.cloud.gov/preview/uswds/uswds-site/accelerator/3940-printer-friendly/documentation/settings/#print-styles